test/system/010-images.bats

```diff
@@ -384,10 +384,17 @@ EOF
     is "${lines[-1]}" "$IMAGE true" "image from readwrite store"
 
     CONTAINERS_STORAGE_CONF=$sconf run_podman images -a -n --format "{{.Id}}"
-    id=${lines[-1]}
+    local cd=$(image_config_digest $IMAGE) # Without $sconf, i.e. from the read-write store.
 
     CONTAINERS_STORAGE_CONF=$sconf run_podman pull -q $IMAGE
-    is "$output" "$id" "pull -q $IMAGE, using storage.conf"
+    local cd2=$(CONTAINERS_STORAGE_CONF=$sconf image_config_digest $IMAGE)
+    assert "$cd2" = "$cd" "pull -q $IMAGE, using storage.conf"
```

Comment:
Note to self: this is failing??
```
expected: = e7cc5fec8db203e6b47f409bd5be067d48ffd3ffbc2e9e0090457553ef5e1def
   actual:   db80d21787bb899c8e4fbb1e630a4d058a99d742318e5db66bbe929dce9ad3b3
```

---

In general it’s unclear to me yet whether this test approach is viable: c/storage additional stores are expected to never go away or to have images removed.

Ordinary teardown after the test failure is failing because we have an image with dangling layers.

Maybe this can be easily fixed up; or maybe the only reasonable approach is to create an entirely separate store for testing this feature.